pkgs/servers/mqtt/mosquitto/default.nix

```diff
@@ -1,36 +1,37 @@
-{ stdenv, fetchurl, openssl, libuuid, cmake, libwebsockets, c-ares, libuv }:
+{ stdenv, lib, fetchurl, openssl, libuuid, cmake, libwebsockets, c-ares, libuv }:
 
 stdenv.mkDerivation rec {
-  pname = "mosquitto";
-  version = "1.4.14";
-
-  name = "${pname}-${version}";
+  name = "mosquitto-${version}";
+  version = "1.4.15";
 
   src = fetchurl {
-    url = "http://mosquitto.org/files/source/mosquitto-${version}.tar.gz";
-    sha256 = "1la2577h7hcyj7lq26vizj0sh2zmi9m7nbxjp3aalayi66kiysqm";
+    url = "http://mosquitto.org/files/source/${name}.tar.gz";
+    sha256 = "10wsm1n4y61nz45zwk4zjhvrfd86r2cq33370m5wjkivb8j3wfvx";
   };
 
-  buildInputs = [ openssl libuuid libwebsockets c-ares libuv ]
-    ++ stdenv.lib.optional stdenv.isDarwin cmake;
+  buildInputs = [ openssl libuuid libwebsockets c-ares libuv ];
+
+  nativeBuildInputs = [ cmake ];
 
-  makeFlags = stdenv.lib.optionals stdenv.isLinux [
-    "DESTDIR=$(out)"
-    "PREFIX="
-  ];
+  enableParallelBuilding = true;
 
   postPatch = ''
     substituteInPlace config.mk \
-      --replace "/usr/local" ""
-    substituteInPlace config.mk \
+      --replace "/usr/local" "" \
       --replace "WITH_WEBSOCKETS:=no" "WITH_WEBSOCKETS:=yes"
+
+    for d in lib lib/cpp src ; do
+      substituteInPlace $d/CMakeLists.txt \
+        --replace /sbin/ldconfig ldconfig
```

Comment:
Oh man, why does debian still no put `/sbin` in the user's PATH. This is just broken.